wiki/Client/File_formats/Osu_(file_format)/en.md

```diff
@@ -31,7 +31,7 @@ The following content is separated into sections, indicated by section titles in
 | `PreviewTime` | Integer | Time in milliseconds when the audio preview should start | -1 |
 | `Countdown` | Integer | Speed of the countdown before the first hit object (`0` = no countdown, `1` = normal, `2` = half, `3` = double) | 1 |
 | `SampleSet` | String | Sample set that will be used if timing points do not override it (`Normal`, `Soft`, `Drum`) | Normal |
-| `StackLeniency` | Decimal | Multiplier for the threshold in time where hit objects placed close together stack (0–1) | 0.7 |
+| `StackLeniency` | Decimal | [Multiplier](/wiki/Beatmap/Stack_leniency) for the threshold in time where hit objects placed close together stack (0–1) | 0.7 |
```

Comment:
bit of a weird spot to put the link but, but i suppose it's due to the wording

Reply:
yeah